target_postgres/tests/test_standard_target.py

```diff
@@ -242,101 +280,49 @@ def test_schema_no_properties(postgres_target):
     singer_file_to_target(file_name, postgres_target)
 
 
-# TODO test that data is correct
 def test_schema_updates(postgres_target):
     file_name = "schema_updates.singer"
     singer_file_to_target(file_name, postgres_target)
+    row = {
+        "id": 1,
+        "a1": Decimal("101"),
+        "a2": "string1",
+        "a3": None,
+        "a4": None,
+        "a5": None,
+        "a6": None,
+    }
+    verify_data(postgres_target, "test_schema_updates", 6, "id", row)
 
 
-# TODO test that data is correct
 def test_multiple_state_messages(postgres_target):
     file_name = "multiple_state_messages.singer"
     singer_file_to_target(file_name, postgres_target)
+    row = {"id": 1, "metric": 100}
+    verify_data(postgres_target, "test_multiple_state_messages_a", 6, "id", row)
+    row = {"id": 1, "metric": 110}
+    verify_data(postgres_target, "test_multiple_state_messages_b", 6, "id", row)
 
 
 def test_relational_data(postgres_target):
-    engine = create_engine(postgres_target)
     file_name = "user_location_data.singer"
     singer_file_to_target(file_name, postgres_target)
 
     file_name = "user_location_upsert_data.singer"
     singer_file_to_target(file_name, postgres_target)
 
-    schema_name = postgres_target.config["default_target_schema"]
+    user = {"id": 1, "name": "Johny"}
+    location = {"id": 1, "name": "Philly"}
+    user_in_location = {
+        "id": 1,
+        "user_id": 1,
+        "location_id": 4,
+        "info": {"weather": "rainy", "mood": "sad"},
+    }
 
-    with engine.connect() as connection:
```

Comment:
I like that you want to consolidate the Tests but I think this test had a pretty solid data test and you removed a lot of the checks that were there before. We could just leave this one alone, or you could expand your verify data function to be able to work with multiple rows of data.

Reply:
Expanded to work with multiple rows.